pkg/api/store/workload/transform_store.go

```diff
@@ -67,6 +67,23 @@ func setTransitioning(apiContext *types.APIContext, data map[string]interface{})
 	if data["transitioning"] == "yes" || data["transitioning"] == "error" {
 		return
 	}
+	containers, ok := data["containers"].([]interface{})
```

Comment:
we can directly use `convert.ToMapSlice`, then no need to convert each container to `map[string]interface{}`